File: JS/IdadeEmDias.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */

let days = parseInt( gets() );
let out = splitAgeDays( days );

console.log( `${out[0]} ano(s)\n${out[1]} mes(es)\n${out[2]} dia(s)` );

/**
 * Split age in days, month and year
 */
function splitAgeDays(days = 0) {
    let year = Math.floor(days / 365);
    let month = Math.floor((days % 365) / 30);
    let day = days % 365 % 30;

    return [year, month, day];
}
module.exports = {splitAgeDays}

File: GO/IdadeEmDias.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import "math"

/**
 * Split age in days, month and year
 */
func splitAgeDays(days int) []int {
	year := math.Floor(float64(days) / 365)
	month := math.Floor((float64(days) - (year * 365)) / 30)
	day := days % 365 % 30

	return []int{int(year), int(month), int(day)}
}
